tests/dummy/app/templates/components-examples/flexberry-objectlistview/inheritance-models/successor-phone-edit.hbs

```diff
@@ -0,0 +1,74 @@
+<h3 class="ui header">{{t "forms.ember-flexberry-dummy-application-user-edit.caption"}}</h3>
```

Comment:
All captions/labels in form are for forms.ember-flexberry-dummy-application-user-edit, but bust be for forms.components-examples.flexberry-objectlistview.inheritance-models.successor-phone-edit